### PR TITLE
Bug 1142087 - [Settings] Update privacy policies

### PR DESCRIPTION
**Pull request by KevinGrandon**
https://bugzilla.mozilla.org/show_bug.cgi?id=1142087

### CODE REVIEW

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=31132e3540e2)

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=325c6a282624)

File: apps/settings/style/icons.css [hunk 1]
```diff
@@ -280,10 +280,6 @@ aside.connecting {
   background-position: 0 0;
 }
 
-.privacy-everythingME:before {
-  background-position: -3.2rem 0;
```
Comment (KevinGrandon) [inline apps/settings/style/icons.css]: The image still exists in the image asset - we can remove it if needed, but would need to get UX involved to generate the new assets I think?

Comment (crh0716) [inline apps/settings/style/icons.css]: Yes, we can request the new asset from Helen Huang.

**Pull request merged**